mythicmaker/src/pages/Registration/Registration.tsx

```diff
@@ -0,0 +1,161 @@
+                  />
+                  <InputError name="confirmPassword" />
+                </div>
+                <CheckField
+                  id="notRobot"
+                  name="notRobot"
+                  label="Nie jestem robotem"
+                />
+                <InputError name="notRobot" />
+                <button
+                  type="submit"
+                  className="px-4 py-2 mt-4 font-semibold text-white rounded-lg font-secondaryFont bg-thirdColor hover:bg-fourthColor focus:outline-none focus:ring-2 focus:bg-fourthColor"
+                  aria-label="Register Account"
+                >
+                  Załóż konto
+                </button>
+              </Form>
+            )}
+          </Formik>
+        </section>
+      </main>
+      <footer className="relative flex flex-col items-center justify-center w-full md:flex-row h-max">
+        <div className="absolute w-[80%] h-[32px] top-[-25px] left-1/2 translate-x-[-50%] bg-white z-10 rounded-2xl">
+          <div className="relative w-full h-full">
+            {[...Array(2)].map((_, index) => (
+              <div
+                key={index}
+                className="h-[50px] w-[50px] rounded-[15px] bg-gray-200 absolute top-1/2"
+                style={{
+                  [index === 0 ? 'left' : 'right']: '-15px',
+                  transform: 'translateY(-50%)',
+                }}
+              >
+                <img
+                  src={iconPlus}
+                  alt="Plus icon for adding features"
+                  className="invert sepia saturate-200"
+                />
+              </div>
+            ))}
+          </div>
+        </div>
+
+        <FeatureCard
+          icon={icon2}
+          altText="Ikona do tworzenia i zarządzania postaciami"
+          title="Twórz i zarządzaj postaciami"
+          description="W MythicMaker możesz łatwo tworzyć konta i projektować swoje unikalne postacie do RPG. Każdy profil zawiera wszystkie niezbędne informacje, takie jak imię, rasa, historia, wygląd i zdjęcie."
+          bgColor="bg-primaryColor"
+        />
+        <FeatureCard
+          icon={icon3}
+          altText="Ikona do rozwijania umiejętności"
+          title="Rozwijaj swoje umiejętności"
+          description="W miarę grania zdobywasz doświadczenie, które pozwala twojej postaci rosnąć. MythicMaker umożliwia dodawanie punktów doświadczenia i modyfikowanie statystyk, takich jak siła, zręczność i inteligencja."
+          bgColor="bg-thirdColor"
+        />
+        <FeatureCard
+          icon={icon1}
+          altText="Ikona do zarządzania ekwipunkiem"
+          title="Zarządzaj swoim ekwipunkiem"
+          description="Zyskaj pełny wgląd w sprzęt, który twoja postać ma na sobie. MythicMaker pozwala ci przeglądać i edytować swój ekwipunek, co ułatwia strategizowanie i zarządzanie postacią."
+          bgColor="bg-secondaryColor"
+        />
+      </footer>
+    </div>
```

Comment:
Czy to nie duplikat z Loginu? Przeniósłbym gdzieś to reużywalnie do innego komponentu